terraform/user.tf

```diff
@@ -0,0 +1,15 @@
+# Create the test user
+module "user" {
+  source = "github.com/cisagov/molecule-iam-user-tf-module"
```

Comment:
If this role doesn't need SSM Parameter Store access then is [cisagov/ci-iam-user-tf-module](https://github.com/cisagov/ci-iam-user-tf-module) a better fit for user creation? There's a lot of extra cruft for unused functionality using this module.

Reply:
I wouldn't call it "a lot" of extra cruft, but there is definitely some.  I think we have a tradeoff here between keeping things consistent with the skeleton vs. creating some extra AWS resources (the SSM policy and role, which are unused but don't cost us anything).  I'm inclined to keep it as is for consistency, but happy to hear other points of view.

Reply:
The "a lot" to me is not just the extra things needed to support it in this repository's configuration, but also the extra resources created in AWS that shouldn't exist like everything around the `/not/used` parameter.

Reply:
I can't remember if there was a reason why we didn't do this in the past, but I will see if I can make a PR in `cisagov/molecule-iam-user-tf-module` so that if the list of SSM parameters is empty, those SSM-related resources will not be created.  How does that sound?

Reply:
Please take a look at https://github.com/cisagov/molecule-iam-user-tf-module/pull/62.  Once that PR is approved and merged, I will update this PR and any other similar PRs whose test users do not need to read SSM parameters.

Reply:
https://github.com/cisagov/molecule-iam-user-tf-module/pull/62 has been merged.  @mcdonnnj can we resolve this conversation?